client/state/data-layer/wpcom/account-recovery/reset/test/index.js

```diff
@@ -1,78 +1,85 @@
 /**
  * External dependencies
  */
-import { assert } from 'chai';
-import sinon from 'sinon';
+import { expect } from 'chai';
+import { spy } from 'sinon';
 
 /**
  * Internal dependencies
  */
 
-import { handleResetPasswordRequest } from '../';
-import useNock from 'test/helpers/use-nock';
+import {
+	resetPassword,
+	handleError,
+	handleSuccess,
+} from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
 
 import {
 	ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS,
 	ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR,
 } from 'state/action-types';
 
-describe( 'handleResetPasswordRequest()', () => {
-	const apiBaseUrl = 'https://public-api.wordpress.com:443';
-	const endpoint = '/wpcom/v2/account-recovery/reset';
+describe( 'account-recovery/reset', () => {
+	describe( '#handleResetPasswordRequest', () => {
+		const dispatchSpy = spy();
+		const dummyAction = {
+			userData: {
+				user: 'foo',
+			},
+			method: 'primary_email',
+			key: 'a-super-secret-key',
+			password: 'my-new-password-which-I-cannot-remember',
+		};
 
-	const params = {
-		userData: {
-			user: 'foo',
-		},
-		method: 'primary_email',
-		key: 'a-super-secret-key',
-		password: 'my-new-password-which-I-cannot-remember',
-	};
+		resetPassword( { dispatch: dispatchSpy }, dummyAction );
```

Comment:
Should this call (and the calls to `handleError` and `handleSuccess` below) be either inside the `it` or in a `beforeEach` function?

Reply:
I followed the pattern here: https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom/me/send-verification-email/test/index.js, but I agree that this should go inside the `it()`. 👍